publications/ohbm_2024/brainhack_project.md

```diff
@@ -0,0 +1,100 @@
+To be submitted [here](https://github.com/ohbm/hackathon2024/issues/new?assignees=sina-mansour&labels=Hackathon+Project&projects=&template=brainhack-hacktrack-project.yml&title=%3CMy+Project+Name%3E)
+
+Combined from previous years:
+- 2023: https://github.com/ohbm/hackathon2023/issues/15
+- 2022: https://github.com/ohbm/hackathon2022/issues/48
+
+# Title
+(Name of your awesome project. Please also update the title of the issue to be the title of your project)
+```txt
+Nilearn: Statistics for Neuroimaging in Python
+```
+
+# Short description and the goals for the OHBM BrainHack
+(Please provide a short description and goals for the hackathon, check [the guide](https://github.com/ohbm/hackathon2024/blob/main/.github/ISSUE_TEMPLATE/handbooks/projects.md#goals) for suggestions)
+```txt
+Nilearn is an open-source Python package for fast and easy analysis and visualization of MRI brain images. It provides statistical and machine-learning tools, with instructive documentation and a friendly community. It includes applications such as multi-voxel pattern analysis (MVPA), decoding, predictive modelling, functional connectivity, and brain parcellations.
+
+Recent work in Nilearn has been centered on developing a new API to allow users to seamlessly work with surface data in a manner similar to volumetric data, enhancing support for the General Linear Model (GLM), enhancing the BIDS interface, and improving and updating the infrastructure and codebase.
+
+To this end, we aim to get contributors to get familiar with the package, improve and provide feedback on the new surface data functionality as well as the package overall. Contributors are encouraged to work on issues in the project board by opening pull requests. New contributors should look for the "Good First Issue" label on the project board to get started.
```

Comment:
What I might suggest -- if you're the only maintainer available at the event, @man-shu -- is to really focus on helping with user-presented issues. So, advertising this as a time for dedicated maintainer availability where you can direct them / you can encourage opening new issues and PRs as necessary to solve presented problems.

@alexisthual and I had done something similar at the Glasgow brainhack, and we certainly had a lot of interest. It ends up being more of a focus on community outreach than on directly advancing the project, but I think that balance might make sense for this year's brainhack. Let me know what you think (or if you'd make a different recommendation, Alexis !)

Reply:
Thanks for the suggestion, @emdupre! Will add some stuff about the outreach thing as well.